### PR TITLE
Added tags for breed/tame items.

### PR DESCRIPTION
**Pull request by Xaidee**
Cherry picked commits from #23 for 1.19

### CODE REVIEW

File: src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java [hunk 1]
```diff
@@ -96,7 +94,7 @@ protected void registerGoals() {
         this.goalSelector.addGoal(9, new GoToWaterGoal(this, 0.8D));
     }
 
-    public static boolean checkGrebeSpawnRules(EntityType<LittleGrebeEntity> animal, LevelAccessor worldIn, MobSpawnType reason, BlockPos pos, RandomSource randomIn) {
+    public static boolean checkGrebeSpawnRules(EntityType<LittleGrebeEntity> animal, LevelAccessor worldIn, MobSpawnType reason, BlockPos pos, Random randomIn) {
```
Comment (Hellion-Games) [inline src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java]: I'm not sure why any changes are being made to this class. It's not relevant to the PR.

**Review by Dariensg — CHANGES_REQUESTED** [review]

File: src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java (hunk 1 above)
Comment (Hellion-Games) [reply src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java]: After looking at the other PR, there is a temptation tag relevant to this class, but you don't make this change here and instead are changing irrelevant code. Probably just some weird mistake with the cherry-picking!

Comment (Xaidee) [reply src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java]: Yes that'll be the case, i thought i checked everything so my bad

File: src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java [hunk 2]
```diff
@@ -59,8 +59,6 @@
 import javax.annotation.Nullable;
 import java.util.UUID;
 
-import net.minecraft.world.entity.AgeableMob.AgeableMobGroupData;
-
 public class LittleGrebeEntity extends Animal implements IAnimatable {
     public static final Ingredient TEMPTATION_ITEMS = Ingredient.of(Items.COD, Items.SALMON, Items.TROPICAL_FISH);
```
Comment (Hellion-Games) [inline src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java]: Still doing a weird thing here with removing an import, but also you didn't actually change the temptation items here to a tag.

**Review by Dariensg — CHANGES_REQUESTED** [review]

File: src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java (hunk 2 above)
Comment (Xaidee) [reply src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java]: Maybe cherry picking wasn't the best way to go about things for this PR lol

Comment (Hellion-Games) [reply src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java]: Hahaha you’re fine! Sorry things have been so slow on this. I’m finally getting to updating things. Basically just waiting on these PRs to be ready!

Comment (Xaidee) [reply src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java]: Okay, now hopefully that's everything fixed

**Review by Dariensg — APPROVED** [review]
LGTM

**Comment by Dariensg** [comment]
Looks good! I'll get these merged and released later today!

**Pull request merged**